src/main/java/core/basesyntax/Lotery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lotery {
+    public static int getRandomBall() {
```

Comment:
- same here about static
- this method should return a Ball object, not a random number